scheduler/config-k8s.edn

```diff
@@ -15,7 +15,8 @@
  :mesos {:leader-path "/cook-scheduler"
          ; because the minikube users and the local users may have different IDs, tests that require file permissions
          ; (like test_default_container_volumes) will fail without this set.
-         :run-as-user "root"}
+         :run-as-user "root"
+         :enable-gpu-support true}
```

Comment:
Might be good to add a comment here about how this is not actually related to mesos.